tests/credential_plugins_test.py

```diff
@@ -137,13 +137,19 @@ class TestDelineaImports:
     library, so these tests are designed to fail if these wind up using the
     fallback import."""
 
-    def test_dsv_import(self):
-        from awx_plugins.credentials.dsv import SecretsVault  # noqa
+    def test_dsv_import(self) -> None:
+        from awx_plugins.credentials.dsv import SecretsVault  # noqa: F401
```

Comment:
This is not related to MyPy/typing. Could you move the change out into a separate PR? That PR will have to also make the CI changes since this update makes https://github.com/ansible/awx-plugins/actions/runs/10818045643/job/30020907062?pr=26 pass, and it needs to move back into the regular pre-commit job.

Reply:
I have removed those codes and have reverted both of these comments back to `# noqa`.

Reply:
Marking this as resolved.